R/data.R

```diff
@@ -0,0 +1,17 @@
+#' Antidepressant trial data.
+#'
+#' A dataset containing the data from a public available antidepressant clinical trial of an active drug versus placebo.
+#' The relevant endpoint is the Hamilton 17-item rating scale for depression (HAMD17) which was assessed at baseline and weeks 1, 2, 4, and 6.
+#' Study drug discontinuation occurred in 24% (20/84) for the active drug and 26% (23/88) for placebo.
+#' All data after study drug discontinuation are missing and there is a single additional intermittent missing observation.
+#'
+#' @format A data frame with 608 rows and 11 variables:
+#'   - `PATIENT`: patients IDs.
+#'   - `VISIT`: post-baseline visit. Has levels 4,5,6,7.
+#'   - `THERAPY`: the treatment group variable. It is equal to `PLACEBO` for observations
+#'   from the placebo arm, or `DRUG` for observations from the active arm.
+#'   - `basval`: baseline outcome value.
+#'   - `HAMDTL17`: Hamilton 17-item rating scale value.
+#'   - `change`: change from baseline in the Hamilton 17-item rating scale.
+#'   - ...
```

Comment:
Howcome `...` should we not document all variables. Also there is a mix of lower case and upper case variables. Is this how the dataset was provided from source ? I would prefer to standardise it to all upper or all lower.   

Additionally we should provide a source url or reference of some description if possible

Reply:
@gowerc the "..." was because there are other variables (some of them baseline covariates, some of them I don't honestly know what they are about exactly) that we are not interested in. Should I try to document all of them?

I did not change the variable names. I took the dataset from the https://www.lshtm.ac.uk/research/centres-projects-groups/missing-data#dia-missing-data (I can add the url) and only made factor or numeric the variables I was interested in, and then created a .RData object.

Reply:
I think we should probs re-format the names to be standardised. I don't think it matters that we change it from the source as long as its just the metadata we are changing and quote the original.  I do think we need to document all variables we include but I also think its fine to drop variables that we don't want to include / document, I've seen several instances where people just include a subset of the full original dataset.  (though if you do subset it make sure that this is documented).

Reply:
Ok, I have mentioned the url of the website where the dataset can be downloaded. Additionally I have documented all the 11 variables of the dataset and standardized the name of the columns (all have now only capital letters).